.github/workflows/test.yml

```diff
@@ -97,8 +97,8 @@ jobs:
         env:
           YARN_CHECKSUM_BEHAVIOR: "update"
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
-      - name: Install Replay Playwright
-        run: npx @replayio/playwright install
+      - name: Install Replay Chromium
+        run: npx replayio install
```

Comment:
this won't work yet because we havent yet released a version with this alias of the `update` command